test/assets/output/preset-mini-targets.css

```diff
@@ -961,7 +961,11 @@ unocss .scope-\[unocss\]\:block{display:block;}
 .box-content{box-sizing:content-box;}
 .box-inherit{box-sizing:inherit;}
 .transition{transition-property:color,background-color,border-color,outline-color,text-decoration-color,fill,stroke,opacity,box-shadow,transform,filter,backdrop-filter;transition-timing-function:cubic-bezier(0.4, 0, 0.2, 1);transition-duration:150ms;}
-.transition-200{transition-property:color,background-color,border-color,outline-color,text-decoration-color,fill,stroke,opacity,box-shadow,transform,filter,backdrop-filter;transition-timing-function:cubic-bezier(0.4, 0, 0.2, 1);transition-duration:200ms;}
+.transition-\[width_height_colors\]{transition-property:width,height,color,background-color,border-color,outline-color,text-decoration-color,fill,stroke;transition-timing-function:cubic-bezier(0.4, 0, 0.2, 1);transition-duration:150ms;}
```

Comment:
This is intentional. Since `colors` has many compose properties, so when using `[xxx]`, you can not only customize the properties, but also use the theme.

Reply:
In our grammar, `_` is converted to spaces, which may cause ambiguity when used on other attributes.